src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -2331,6 +2331,21 @@
     /**
      * The {@code |} (bitwise OR) operator.
      *
+     * <p>For example:</p>
+     * <pre>
+     * a = a | b;
```

Comment:
scope of this issue is only `MINUS` token, you do not need to do other tokens

Reply:
The MINUS part is right yet some checks are failing, mvn verify says build successful, what to do?

Reply:
There is only one commit, I have rebased and squashed the commits but yet it fails?
How to solve that, I have no clue and the code is perfect?

Reply:
please check your `shubham11941140:master` it says `This branch is 1 commit ahead, 41 commits behind checkstyle:master.`

fetch latest updates from checkstyle master and rebase on it